app/src/main/java/com/ultratechies/ghala/ui/auth/model/RegistrationUserDetails.kt

```diff
@@ -0,0 +1,10 @@
+package com.ultratechies.ghala.ui.auth.model
+
+data class RegistrationUserDetails(
```

Comment:
would it be easier, structurally to have the models in data/models package? @Njumbi

Reply:
Essentially it will be like that, once we have a working backend will refactor.